pkg/frontend/openshiftcluster_putorpatch.go

```diff
@@ -245,6 +245,9 @@ func (f *frontend) _putOrPatchOpenShiftCluster(ctx context.Context, log *logrus.
 	doc.OpenShiftCluster.Properties.ClusterProfile.PullSecret = ""
 	doc.OpenShiftCluster.Properties.ServicePrincipalProfile.ClientSecret = ""
 
+	// We don't return enriched worker profile data on PUT/PATCH operations
+	doc.OpenShiftCluster.Properties.WorkerProfilesStatus = nil
```

Comment:
If it's a read-only property, why not?

Reply:
I'm fine doing that, but it was decided to return the _WorkerProfilesStatus_ field only under GET/LIST operation during design meeting.